src/transformers/commands/pt_to_tf.py

```diff
@@ -270,9 +270,13 @@ def run(self):
 
         # Load models and acquire a basic input compatible with the model.
         pt_model = pt_class.from_pretrained(self._local_dir)
+        pt_model.eval()
```

Comment:
If you don't get a model in `eval` mode with `from_pretrained`, then it's a bug.
(Not saying we shouldn't be safe, but this line shouldn't be necessary.)